src/ui/public/agg_types/__tests__/agg_type.js

```diff
@@ -94,6 +92,7 @@ describe('AggType Class', function () {
           });
 
           let aggConfig = vis.aggs.byTypeName.date_histogram[0];
+          const aggType = aggConfig.type;
 
           expect(aggType.getFormat(aggConfig)).to.be(fieldFormat.getDefaultInstance('date'));
```

Comment:
Not sure I understand the changes in this file, tests seem to pass without them?